backend/shelf-server/src/main/java/com/htec/shelfserver/controller/ConfirmationTokenController.java

```diff
@@ -0,0 +1,23 @@
+package com.htec.shelfserver.controller;
+
+import com.htec.shelfserver.responseModel.UserResponseModel;
+import com.htec.shelfserver.service.ConfirmationTokenService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/users/register")
+@AllArgsConstructor
+public class ConfirmationTokenController {
+
+    private final ConfirmationTokenService confirmationTokenService;
+
+    @GetMapping(path = "/confirm")
+    public ResponseEntity<String> confirm(@RequestParam("token") String token) {
```

Comment:
general rule is that we shouldn't use verbs in paths (we should use nouns). Also do we get something with this endpoint or we are performing action? Maybe GET is not so good method choice here